src/main/java/org/mitre/fhir/authorization/ServerConformanceWithAuthorizationProvider.java

```diff
@@ -106,4 +114,14 @@ public CapabilityStatement getServerConformance(HttpServletRequest theRequest,
 
     return capabilityStatement;
   }
+
+  private Extension createOAuthUriExtension(String extensionUrl, String extensionValue) {
+    Extension extension = new Extension();
+    extension.setUrl(extensionUrl);
+    UriType value = new UriType();
+    value.setValue(extensionValue);
+    extension.setValue(value);
```

Comment:
```suggestion
    Extension extension = new Extension(extensionUrl, new UriType().setValue(extensionValue));
```

[It looks like there is a constructor for Extension that accepts the URL and value](https://hapifhir.io/hapi-fhir/apidocs/hapi-fhir-structures-r4/org/hl7/fhir/r4/model/Extension.html#%3Cinit%3E(java.lang.String,org.hl7.fhir.instance.model.api.IBaseDatatype)) that could shorten this a bit.  Tested it out locally with `docker-compose up --build` and it seemed to work fine.

Reply:
This method isn't needed at all now since it's essentially just duplicating the functionality of the constructor.